flink-core/src/main/java/org/apache/flink/configuration/NettyShuffleEnvironmentOptions.java

```diff
@@ -174,6 +174,19 @@
 				" help relieve back-pressure caused by unbalanced data distribution among the subpartitions. This value should be" +
 				" increased in case of higher round trip times between nodes and/or larger number of machines in the cluster.");
 
+	/**
```

Comment:
Could you also update in the L159 description a bit? I think after we dropped the non credit based code path, it is a bit out of date.

> Number of exclusive network buffers to use for each outgoing/incoming channel (subpartition/input channel) in the credit-based flow control model. It should be configured at least 2 for good performance. 1 buffer is for receiving in-flight data in the subpartition and 1 buffer is for parallel serialization.

Otherwise, as it is, it is a bit confusing with the new description.